src/components/content/credential/Credential.tsx

```diff
@@ -0,0 +1,226 @@
+    const [credentialDefinitionList, setCredentialDefinitionList] = useState<CredentialVariables[]>([]);
+    const [credentialDetails, setCredentialDetails] = useState<CredentialVariable[]>([]);
+    const [createCredential, setCreateCredential] = useState<CreateCredential>({
+        csp: '' as CredentialVariables.csp,
+        description: '',
+        name: '',
+        timeToLive: 0,
+        type: '' as CredentialVariables.type,
+        variables: [],
+        xpanseUser: '',
+    });
+
+    const columns: ColumnsType<CredentialVariables> = [
+        {
+            title: 'Csp',
+            dataIndex: 'csp',
+        },
+        {
+            title: 'User',
+            dataIndex: 'xpanseUser',
+        },
+        {
+            title: 'Name',
+            dataIndex: 'name',
+        },
+        {
+            title: 'Type',
+            dataIndex: 'type',
+        },
+        {
+            title: 'Description',
+            dataIndex: 'description',
+        },
+        {
+            title: 'Operation',
+            dataIndex: 'operation',
+            render: (text: string, record: CredentialVariables) => {
+                return (
+                    <>
+                        <Space size='middle'>
+                            <Button type='primary' icon={<FullscreenOutlined />} onClick={() => details(record)}>
+                                Details
+                            </Button>
+                            <Popconfirm
+                                title='Delete the Credential'
+                                description='Are you sure to delete the Credential?'
+                                okText='Yes'
+                                cancelText='No'
+                                onConfirm={() => deleteCredential(record)}
+                            >
+                                <Button type='primary' icon={<MinusCircleOutlined />}>
+                                    Delete
+                                </Button>
+                            </Popconfirm>
+                            <Button
+                                type='primary'
+                                icon={<InfoCircleOutlined />}
+                                onClick={() => updateCredential(record)}
+                            >
+                                Update
+                            </Button>
+                        </Space>
+                    </>
+                );
+            },
+        },
+    ];
+
+    const addCredential = () => {
+        setIsAddOpen(true);
+    };
+
+    const deleteCredential = (credentialVariables: CredentialVariables) => {
+        void CredentialsManagementService.deleteCredential(
+            credentialVariables.csp,
+            credentialVariables.xpanseUser,
+            credentialVariables.type
+        )
+            .then(() => {
+                getTipInfo('success', 'delete success!');
```

Comment:
Message -  Deleting Credential Successful
Also the alert should have margins.

![image](https://github.com/eclipse-xpanse/xpanse-ui/assets/54129659/20d68c1b-e540-469f-aa03-4a27fc492887)